btree/btree.c

```diff
@@ -1143,12 +1145,15 @@ static uint32_t         trees_loaded = 0;
 static struct m0_rwlock trees_lock;
 
 #ifndef __KERNEL__
-static void node_init(struct node_op *n_op, int ksize, int vsize)
+static void node_init(struct node_op *n_op, int ksize, int vsize,
+		      struct m0_be_tx *tx)
 {
 	const struct node_type *n_type = n_op->no_node->n_type;
 
 	n_type->nt_init(n_op->no_node, segaddr_shift(&n_op->no_node->n_addr),
 			ksize, vsize);
+	if (tx != NULL)
+		mem_update(n_op, tx, n_op->no_node, sizeof *n_op->no_node);
```

Comment:
Again it is unwise to call the mem_* functions directly from node functions. You can use the structure of function pointers to reach the correct functions.

Reply:
mem_update is not among the mem_* functions.
It is just a generic name I gave.

Reply:
The function will be responsible to call the m0_be_tx_capture() from our new btree